include/zim/search_iterator.h

```diff
@@ -56,7 +56,7 @@ class SearchIterator : public std::iterator<std::bidirectional_iterator_tag, Ent
         int getScore() const;
         std::string getSnippet() const;
         int getWordCount() const;
-        int getSize() const;
+        DEPRECATED int getSize() const;
```

Comment:
This comment pertains to the commit message.

"Depreciate" is a valid word/verb in English and it is different from "deprecate".